src/lib/routes/admin-api/bootstrap-ui.ts

```diff
@@ -71,15 +87,30 @@ class BootstrapController extends Controller {
         this.featureTypeService = featureTypeService;
         this.emailService = emailService;
         this.versionService = versionService;
-
-        this.logger = config.getLogger(
-            'routes/admin-api/bootstrap-controller.ts',
-        );
-
-        this.get('/', this.bootstrap);
+        this.openApiService = openApiService;
+
+        this.logger = config.getLogger('routes/admin-api/bootstrap-ui.ts');
+        this.route({
+            method: 'get',
+            path: '',
+            handler: this.bootstrap,
+            permission: NONE,
+            middleware: [
+                openApiService.validPath({
+                    tags: ['other'],
+                    operationId: 'bootstrapUiSchema',
```

Comment:
Would give the handler and operationId the same name, maybe `getBootstrapUi` or `getUiBootstrapData`.